contrib/pg_tde/documentation/docs/architecture/index.md

```diff
@@ -1,13 +1,15 @@
 # Architecture
 
-`pg_tde` is a **customizable, complete, data at rest encryption extension** for PostgreSQL.
+This topic outlines the long-term architectural goals and intended design of `pg_tde`. While many components are already implemented, some features are still under active development and may not yet be available in the current release.
```

Comment:
Is this true? I would say not. The bulk of this document should be about current architecture and then we should have a section about the future goals.